flybywiresim/aircraft
# fix(efb): EFB fuel fill button color spill

Fixes #8079 

## Summary of Changes

Fixes the background overflow in the Fuel Page of the EFB.

## Screenshots (if necessary)


Before:
![image](https://github.com/flybywiresim/aircraft/assets/56562388/179b35bc-b4cb-4911-b5bd-76c8a1b7d9f3)

After:
![image](https://github.com/flybywiresim/aircraft/assets/56562388/37174b4e-4866-4887-9df0-b2c2c6c1d32d)


## References





## Additional context



Discord username (if different from GitHub): Micah (micahbcode)

## Testing instructions

1. Start new flight
2. Goto EFB -> Ground -> Fuel
3. Check there is no overflow of the refueling start-/stop


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
